assets/sass/_api-nodejs.scss

```diff
@@ -76,9 +76,10 @@ ul.api {
                 }
             }
 
-            &.datasource {
+            &.datasource, // TODO: Remove `datasource` when it's no longer referenced anywhere.
```

Comment:
Ha I did this in a PR branch I am working on as well.

Reply:
Ah, sorry! Thought this would be a tiny thing I could do, but you're already on top of it! 😬

Reply:
No I absolutely appreciate it! Thank you so much of thinking to do this. :)